Userland/Games/Flood/main.cpp

```diff
@@ -73,11 +73,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(Core::System::unveil("/res", "r"));
     TRY(Core::System::unveil(nullptr, nullptr));
 
-    size_t board_rows = Config::read_i32("Flood"sv, ""sv, "board_rows"sv, 16);
-    size_t board_columns = Config::read_i32("Flood"sv, ""sv, "board_columns"sv, 16);
+    size_t board_rows = Config::read_u32("Flood"sv, ""sv, "board_rows"sv, 16);
+    size_t board_columns = Config::read_u32("Flood"sv, ""sv, "board_columns"sv, 16);
 
-    Config::write_i32("Flood"sv, ""sv, "board_rows"sv, board_rows);
-    Config::write_i32("Flood"sv, ""sv, "board_columns"sv, board_columns);
+    Config::write_u32("Flood"sv, ""sv, "board_rows"sv, board_rows);
```

Comment:
Are you sure size_t and unsigned int are the same? If not, I suggest changing types of these variables to unsigned int in all the files in the Flood directory. Otherwise, feel free to leave it as is.